tox.ini

```diff
@@ -14,23 +14,25 @@ DJANGO_SETTINGS_MODULE = settings
 norecursedirs = .* docs requirements
 
 [testenv]
-setenv =
+setenv = 
     PYTHONPATH = {toxinidir}
 deps =
-    django32: Django>=3.2,<4.0
+    setuptools
```

Comment:
This shouldn't be needed for `Python 3.11` support since `setuptools` was removed in `Python 3.12`.